docs/go.sum

```diff
@@ -1,2 +1,4 @@
 github.com/nginxinc/nginx-hugo-theme v0.41.8 h1:l9Lsl9NMaTmNCSMa/W9sdqwhlKFxfb+6ue6w6+bsMC0=
```

Comment:
I think if you do a `go mod tidy` it will get rid of the old dependency

Reply:
I'll give it a try: I genuinely know nothing about Go tooling outside of my incidental interactions due to Hugo.